src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  display: flex;
+}
+
+.menu li:first-child a {
+  margin-left: 0;
+}
+
+.menu li:last-child a {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  display: flex;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  transform-origin: bottom left;
+}
+
+.link:hover::after {
+  background-color: rgba(0, 172, 220, 1);
+}
+
+.container_logo {
+  width: 40px;
+}
+
+.free {
+  flex-grow: 1;
+}
+
+/* ====== Product cards ====== */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: $stars-size;
+  height: $stars-size;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card__info {
+  text-align: left;
+  padding: 0 16px;
+}
+
+.card__imac-img {
+  margin-top: 32px;
+  margin-bottom: 36px;
+  width: 160px;
+  height: 134px;
+  padding: 0 19px;
+}
+
+.card__title {
+  color: #060b35;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__text-code {
+  color: #616070;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  color: #060b35;
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-name {
+  color: #616070;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__price-number {
+  color: #060b35;
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+}
+
+.card__button-link {
+  text-decoration: none;
+}
+
+.card__button-buy {
+  display: block;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  color: #fff;
+  background-color: #00acdc;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px 0;
+  cursor: pointer;
+  line-height: 40px;
+  border: none;
+  box-sizing: border-box;
+}
```

Comment:
The `.card__button-buy` has a `cursor: pointer` style, which is good for indicating that it's clickable, but make sure that this element is a `<button>` or an `<a>` with an appropriate `href` attribute to be accessible and semantic.